### PR TITLE
Updating README to reflect reality.

### PR DESCRIPTION
**Pull request by mikeal**
Added some notes about the CTC/TSC's lack of involvement in the day-to-day of this repo and steps toward making any of these threads part of the project.

### CODE REVIEW

**Comment by netpoetica** [comment]
I would definitely merge this ;) I was reading through some of the very weighty issues in this repo thinking that a general overview of the intent would cure many stomach knots :-p

**Comment by mikeal** [comment]
@netpoetica I intend to, just want to give people in other timezones time to review :)

**Comment by jasnell** [comment]
LGTM

**Comment by bnoordhuis** [comment]
LGTM

**Comment by rvagg** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
-Next Generation JavaScript IO Platform
+Forum for discussions about what the next generation of Node.js might look like.
+
+This repository is strictly for discussions about what people think Node.js might look like in the future.
+
+The Node.js project's governing bodies (CTC and TSC) tend to stay out of these discussions and allow those
+passionate about discussing these topics to enjoy using this forum.
+
+No particular thread is endorsed by the Node.js project as being part of any roadmap. If a discussion
+reaches a point of maturity it is expected to end up as a new module, an
```
Comment (Trott) [inline README.md]: Nit: `If a discussion reaches a point of maturity, then it is expected...`

(comma and `then` for clarity)

**Pull request merged**